### PR TITLE
Ibm guardium modeling

### PR DESCRIPTION
**Pull request by ellopez777**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-8094](https://jira-dc.paloaltonetworks.com/browse/CIAC-8094)

## Description
This PR contains IBM Guardium mapping and modeling

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/IBMGuardium/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   **Alert Per Time Granularity** - A notification is provided per your selected time frame.
+7. From the **Message Template** list, select XXX
+8. From **Notification Type**, select **SYSLOG**.
+9. Click **Add**, then click **Apply**.
+10. Click **Save**.
+11. Repeat this process for all rules within the policy that you want to forward to Cortex XSIAM Broker VM.
+
+
+### Installing an IBM Guardium policy
+Procedure
```
Comment (ShirleyDenkberg) [inline Packs/IBMGuardium/README.md:59]:
```suggestion

```

File: Packs/IBMGuardium/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+1. Click the **Administration Console** tab.
+2. From the left navigation, select **Configuration** &rarr; **Policy Installation**.
+3. From the Policy Installer pane, select a policy that you created in previous step.
```
Comment (ShirleyDenkberg) [inline Packs/IBMGuardium/README.md:62]:
```suggestion
3. From the Policy Installer pane, select a policy that you created in the previous step.
```

File: Packs/IBMGuardium/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+4. From the **drop-down** list, select **Install and Override**.
```
Comment (ShirleyDenkberg) [inline Packs/IBMGuardium/README.md:63]:
```suggestion
4. From the drop-down list, select **Install and Override**.
```

**Comment by ShirleyDenkberg** [comment]
@OBavly Doc review completed.

**Comment by ShirleyDenkberg** [comment]
@ellopez777 I already reviewed these docs.

**Comment by ellopez777** [comment]
Hi @ShirleyDenkberg , yes I know but I made some changes and thought you would like to review them.

**Comment by ShirleyDenkberg** [comment]
@ellopez777 Looks good.

**Review by OBavly — APPROVED** [review]

**Pull request merged**